### DISCUSSION TITLE
How change IOS screen format?

### DISCUSSION THREAD

**Discussion by ZhorovEgor** [airsdk/Adobe-Runtime-Support#3220] [General]
Howdy! After deploy project on testing, i see this screen size, how can i change screen format? I have checkbox on "Full screen"
![Изображение WhatsApp 2024-05-02 в 15 49 27_492e3457](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/b03567e0-fa7b-4d1d-b841-40902f17332b)
![Screenshot_1](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/315d30ba-1905-4259-b92d-2a4f83081bb7)

**Comment by ajwfrost** [comment]
Screen sizes on iOS are picked up either from the launch storyboard, or from the launch images.. if you've got a commercial AIR SDK license then please create a launch storyboard for the app, see https://docs.airnativeextensions.com/docs/tutorials/ios-icons-assets-car/

thanks

**Reply by ZhorovEgor** [reply]
I'm sorry, but as I studied the information, the Assets.car file contains the application icons, but not the screens themselves. I created Assets.car through one resource (https://www.кхар.com/appicon.html ) and everything was pulled up automatically there. As you know, it is impossible to edit such a file, maybe there are other reasons?

**Reply by ajwfrost** [reply]
Does your application have a launch storyboard? i.e. when it starts up, what do you see first? if there's a Harman/AIR launch screen then this is our default launch storyboard and we'd need to check why this isn't working with the handset model/resolution..

Assets.car contains the app icons and also assets that are required for a launch storyboard...

Also can you please confirm the handset model and the iOS version because Apple do keep on changing how all this works..

**Reply by ZhorovEgor** [reply]
We have a license ( alt.llc) in the lib folder. When we start, we see an animation of our program, it also has cropped edges at the top and bottom.

As for Assets.car, as you can see in the 2nd screenshot, I have a manual file addition at the bottom. Maybe you should add any folders containing images along with Assets.car?
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/724d7d49-9f97-4c4a-b3b2-7c2c9902a24a)

**Reply by ZhorovEgor** [reply]
Hello! I still haven't solved my question, please tell me if there are any other recommendations?

**Reply by FliplineStudios** [reply]
Did you set up a LaunchScreen storyboard?  This is separate from setting up icons in Assets.car (though the images you use in the storyboard will be included in the Assets.car file).  You can find out more here:

https://docs.airnativeextensions.com/docs/tutorials/ios-launchscreens

Most devices need that LaunchScreen storyboard to be able to fill the screen, otherwise it adds black borders to the edges like you're seeing.

**Reply by ZhorovEgor** [reply]
See what the deal is: I created Assets.cars through the website https://www.kxapp.com/appicon.html perhaps the Launch Screen is not initialized there (well, or how it is done and why LaunchScreen is needed, I am not in the course). I tried to create a donor project through Xcode to get Assets.car out of there, but the program when publishing (Adobe Animate) did not accept Assets.car, which was created in Xcode, however, it accepted that it was created through the site. What can be done in this case?

**Reply by ZhorovEgor** [reply]
By the way, i remember this: how i know, the string "Launch Screen" position on Info.plist file. But when i want include Info.plist file in Adobe Animate AIR project, i saw the error, some like "Info.plist is reserved name, please choise another file name"

**Reply by FliplineStudios** [reply]
It looks like that website just creates the icons and no launch screen, from back when icons were the only requirement from Apple.

Have you tried the different methods in those other links?  They explain how to create the icons and launch screen using the command line or in Xcode, and how to add them both to your AIR project (the launch screen info needs to be added to the InfoAdditions section of your application descriptor XML, and also make sure it's packaged into the IPA).

**Reply by ZhorovEgor** [reply]
Good afternoon, everyone. I took a screenshot of 3 different windows. 1. The simulator window where there is a clear problem with the screen resolution. 2 Setting up publishing in Adobe Animate, where you can see that the program setting is satisfactory for correct resolution. 3 The XML file on the basis of which the info.plist is formed, as well as the settings are all entered correctly. What could be the problem? 
![Изображение WhatsApp 2024-06-14 в 11 38 48_6d912f5f](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/706b4afa-690d-4325-8cb9-20b81bd2358b)

**Reply by ZhorovEgor** [reply]
@ajwfrost can you're help with it?

**Reply by FliplineStudios** [reply]
I don't see a LaunchScreen.storyboardc in your Included Files?  The Launch Screen is the _only_ way that iOS determines which resolutions your app supports, so if you're not including it, you will only ever see that default letterboxed screen resolution that you see in the simulator.

**Reply by ZhorovEgor** [reply]
Hello! The situation is as follows: I created a Launch Screen file from an empty project in the Xcode environment, imported it into the project and published a Wisdom.ipa file based on it. But the image is not set up anyway, I need help
![Изображение WhatsApp 2024-07-13 в 22 07 48_6398ce7e](https://github.com/user-attachments/assets/a732ed3f-c93b-4684-abaf-2747bac5ac97)
![Изображение WhatsApp 2024-07-13 в 22 07 48_b7ef6cc1](https://github.com/user-attachments/assets/ebe211d9-8827-4a28-8193-281b2b470efc)
![Изображение WhatsApp 2024-07-13 в 22 07 48_e8784368](https://github.com/user-attachments/assets/03db1ece-b672-4fa9-b570-566643cf20a6)

**Reply by ZhorovEgor** [reply]
Do i need to configurate Launch Screen in xCode before? @FliplineStudios @ajwfrost

**Reply by marchbold** [reply]
Did you specify the launch screen in your info additions? i.e. add the following:

```xml
<key>UILaunchStoryboardName</key>
<string>Launch Screen</string>
```

Note: I haven't tried one with a "space" in the name so if this doesn't work, try renaming the launch screen and removing the space character.

**Comment by ZhorovEgor** [comment]
@FliplineStudios @ajwfrost
